Freshli/LibYearCalculator.cs

```diff
@@ -38,7 +38,12 @@ IManifest manifest
         }
         catch (Exception e) {
           _logger.Warn($"Skipping {package.Name}: {e.Message}");
-          result.Add(package.Name, package.Version, DateTime.MinValue,
+          result.Add(
+            package.Name,
+            null,
```

Comment:
Let's use a named parameter here so that it's more clear what `null` is being passed into.